crates/noirc_frontend/src/ast/structure.rs

```diff
@@ -1,6 +1,7 @@
 use std::fmt::Display;
 
-use crate::{Ident, NoirFunction, Path, UnresolvedType};
+use crate::{Ident, NoirFunction, UnresolvedGenerics, UnresolvedType};
+use iter_extended::vecmap;
 use noirc_errors::Span;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
Should the `generics` fields in `NoirStruct` also be switched to `UnresolvedGenerics` for consistency?